doc/installation.rst

```diff
@@ -143,53 +225,91 @@ Install `Docker Desktop for Mac
     tab and then use the slider to select 8 GB of RAM. Docker Desktop will
     restart and then you will be ready to go.
 
-Using Pre-built Docker Images
+.. _prebuilt_docker:
+
+Using pre-built Docker images
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We provide a `Docker image <https://hub.docker.com/repository/docker/zeekurity/spicy>`_
-with Spicy built from trunk.
+We provide the following Docker images `on DockerHub <https://hub.docker.com/repository/docker/zeekurity/spicy>`_:
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+    :align: center
+
+    * - Spicy Version
+      - Image name
+      - Source
 
-To run the image execute the following command::
+    * - Release
+      - *(coming soon)*
+      - *(coming soon)*
 
-    docker run -it zeekurity/spicy:latest
+    * - Development
+      - ``zeekurity/spicy-dev``
+      - `Dockerfile <https://github.com/zeek/spicy/blob/master/ci/Dockerfile.dockerhub>`_
 
-Spicy is installed in `/opt/spicy`.
+To run the release image, execute the following command::
 
-Build Your Own Spicy Container
+    # docker run -it zeekurity/spicy:latest
+
+These images pre-install the :ref:`Zeek plugin <zeek>` and the `Zeek
+analyzer collection <https://github.com/zeek/spicy-analyzers>`_ as
+well.
+
+Spicy is installed in ``/opt/spicy`` on these images. The development
+image is updated nightly.
+
+.. todo::
+
+    - Switch the nightly image to the `spicy-dev` label.
+
+.. _dockerfiles:
+
+Build your own Spicy container
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can build your Spicy container from one of the Docker files coming
-with Spicy: Go into Spicy's ``docker`` directory and run ``make`` to
-see the container platforms available::
+You can build base images for your own Spicy setup through the `Docker
+files <https://github.com/zeek/spicy/tree/master/docker>`_ coming with
+the distribution. These images do *not* include Spicy itself, just the
+dependencies that it needs on each platform, both for a source build
+and for the using the corresponding binary packages. (The images do
+install the current Zeek version.)
+
+To build an image, go into Spicy's ``docker`` directory and run
+``make`` to see the container platforms available::
 
-    # cd spicy
+    # cd docker
     # make
 
     Run "make build-<platform>", then "make run-<platform>".
 
     Available platforms:
 
         alpine-3.12
+        centos-7
         centos-8
```

Comment:
centos-8 is mentioned here, while above is mentioned:

    * - CentOS 9
      - *(coming soon)

but up there is no mention of CentOS 8, and here no mention of centos-9